src/components/PressCard/index.tsx

```diff
@@ -0,0 +1,79 @@
+import Image from "next/image"
```

Comment:
this looks almost identical to the origin protocol card on storybook, with a little different spacing - do we need both styles?

Reply:
I'll merge this for now but if it's mostly the same component I'd like to DRY it up later.

Reply:
I made the whole Card component clickable, which doesn't really apply here, but I guess they can be combined